### DISCUSSION TITLE
Yape - Chrome Extension

### DISCUSSION THREAD

**Discussion by RemiRigal** [pyload/pyload#4018] [Announcements]
Hello everyone ! I've had mixed feelings about all the available browser extensions for PyLoad (including ClickNLoad etc...), so I've decided to create my own, named... **Yape**. It's an acronym that stands for **Yet Another PyLoad Extension**.

It is dead simple and is supposed to be that way, it features:
- One-click downloads
- Current downloads monitoring (progress, ETA)
- Global bandwidth usage monitoring & one-click speed limiter

It uses the core API of PyLoad so no plugin is needed.

It's a WIP so feedbacks are welcome, as well as any contributions. Thanks !

GitHub: https://github.com/RemiRigal/Yape

**Comment by Unrepentant-Atheist** [comment]
Bruh, this needs to be on firefox :(

**Reply by RemiRigal** [reply]
I'm working on it, it will be available soon, I'll let you know :)

**Reply by RemiRigal** [reply]
Hi @Unrepentant-Atheist !
The Firefox version of Yape is now available, you can get it [here](https://addons.mozilla.org/fr/firefox/addon/yape/). Let me know if you have any issue, and feel free to post a review on the store, thanks ! :)
